src/components/EventList/components/EventItem/index.jsx

```diff
@@ -47,6 +53,14 @@ const EventItem = ({ event }) => {
               timeZone="currentBrowser"
               description={event.Description}
               onClick={() => setExpanded((e) => !e)}
+              lightMode="system" // Set to system @TODO use localstorage setting to update
```

Comment:
This will at least be accurate to the user's preference when the component renders, although it still won't cause the component to re-render.

```suggestion
              lightMode={localStorage.getItem('mui-mode')}
```

Reply:
Yeah, although the component re-renders a lot currently, since it's inside an event or a class on the schedule panel, so if the user closes the event/class and reopens it the button will switch to the correct color.  So it's possible the component will be in the wrong setting immediately after the setting changes, but only while the user stays focused on the component it's in.
I've implemented it and tested it.  In the relatively rare case that a user changes their system setting with one of the add-to-calendar buttons open, the buttons will be in the wrong mode until they get rendered again, which I think is fine, since simple navigation or a page reload will fix it quickly.  Plus, even in the wrong mode the buttons are still perfectly functional, and don't look too terrible.